### DISCUSSION TITLE
Cannot get frigate to work properly after running the helper script

### DISCUSSION THREAD

**Discussion by shwarnock** [tteck/Proxmox#3769] [Help Needed]
### Please verify that you have read and understood the guidelines.

Yes

### A clear and concise description of the issue.

I have run the Frigate Proxmox Helper script in order to create a frigate VM. After running it, I am able to connect to the frigate GUI. However, I am unable to get my camera to show up. I have my streams added to the go2rtc section of the config file, but go2rtc endpoint doesn't return anything.

I spoke with people on the frigate github, and they suggested something is going wrong with the helper script. Any assistance would be appreciated!

### What settings are you currently utilizing?

Advanced Settings

### Which Linux distribution are you employing?

Debian 11

**Comment by tteck** [comment]
> Cannot get frigate to work properly after running the helper script

This appears to be an operational issue rather than a problem with the script itself. Many users have Frigate running successfully without any issues using the provided script.

**Comment by mage-afk** [comment]
I have the same problem. After installing and starting the LXC, frigate does not receive any streams from go2rtc. After restarting go2rtc (systemctl go2rtc restart) everything works fine until the next restart of the LXC. 
If go2rtc is not working, the streams are not in the config of go2rtc. You can check this in the go2rtc web interface (port:1984).

**Reply by abhayachauhan** [reply]
+1 on the same issue. Frigate doesn't seem to be updating the Go2RTC configuration until the go2rtc process is restarted.
